docs/source/release_notes/release-notes.rst

```diff
@@ -83,6 +83,10 @@ develop
            Previously, these were stored as separate entries, meaning that unnecessary values may have been written to the coordination store; this does not affect correctness, but is unperformant.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3733>`__)
 
+    *    - |changed|
+         - Timelock will now no longer create it's high level paxos directory at configuration de-serialization time. Instead it waits until creating each individual learner or acceptor log directory,
+           allowing timelock to rely more accurately on directory existence as a proxy for said timelock node being new or not.
```

Comment:
probably should be `|improved|`, also we normally put the PR number. I'll change this